app/main.py

```diff
@@ -1,8 +1,48 @@
 class Car:
-    # write your code here
-    pass
+
+    def __init__(self, comfort_class: int, clean_mark: int, brand: str):
+        self.comfort_class = comfort_class
+        self.clean_mark = clean_mark
+        self.brand = brand
 
 
 class CarWashStation:
-    # write your code here
-    pass
+
+    def __init__(
+            self,
+            distance_from_city_center: float,
+            clean_power: int,
+            average_rating: float,
+            count_of_ratings: int
+    ):
+        self.distance_from_city_center = distance_from_city_center
+        self.clean_power = clean_power
+        self.average_rating = average_rating
+        self.count_of_ratings = count_of_ratings
+
+    def serve_cars(self, list_cars: list):
+        income_list: list = []
+        for car in list_cars:
+            if car.clean_mark < self.clean_power:
+                income_list.append(self.calculate_washing_price(car))
+                self.wash_single_car(car)
+        return round(sum(income_list), 1)
+
+    def calculate_washing_price(self, car: Car):
+        result: float = 0.0
+        result = (car.comfort_class * (self.clean_power - car.clean_mark))\
+            * self.average_rating / self.distance_from_city_center
+        return round(result, 1)
+
+    def wash_single_car(self, car: Car):
+        if self.clean_power > car.clean_mark:
+            car.clean_mark = self.clean_power
+
+    def rate_service(self, rating: float):
+        self.count_of_ratings += 1
+        result: float = 0.0
+        sum_of_ratings: float = 0.0
```

Comment:
What do you need these two lines for? You can write calculations to them directly